lib/slack.js

```diff
@@ -58,10 +58,10 @@ export default class SlackData extends EventEmitter {
       return this.retry();
     }
 
-    let total = users.length;
+    let total = users.length - 1;
```

Comment:
@jomo simpler, yes.

But let's say Slack stops sending `slackbot` in the users array, then all calculations will be off by 1.